### PR TITLE
[IMP] account: disable quick create on account.account

### PR DESCRIPTION
**Pull request by william-andre**
Task 1958439

Purpose: prevent the erroneous account creation

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/analytic/views/analytic_account_views.xml [hunk 1]
```diff
@@ -282,7 +282,7 @@
                     <group>
                         <group name="analytic_entry" string="Analytic Entry">
                             <field name="name"/>
-                            <field name="account_id"/>
+                            <field name="account_id" options="{'no_create': True}"/>
```
Comment (qdp-odoo) [inline addons/analytic/views/analytic_account_views.xml]: this is an analytic account, if I'm not wrong... so we probably don't want to restrict its creation here

File: addons/analytic/views/analytic_account_views.xml [hunk 2]
```diff
@@ -29,7 +29,7 @@
                         <field name="analytic_distribution_ids" nolabel="1" widget="one2many_list"
                             attrs="{'invisible': [('active_analytic_distribution', '=', False)]}" groups="analytic.group_analytic_accounting">
                             <tree string="Analytic Distribution" editable="bottom">
-                                <field name="account_id"/>
+                                <field name="account_id" options="{'no_create': True}"/>
```
Comment (qdp-odoo) [inline addons/analytic/views/analytic_account_views.xml]: this is an analytic account, if I'm not wrong... so we probably don't want to restrict its creation here

**Comment by fmdl** [comment]
It could be nice to allow or not quick-create on the user preférence, not ?
For demo quick create is great, but in production it is counter productive.

**Comment by william-andre** [comment]
@avw-odoo

**Comment by william-andre** [comment]
@qdp-odoo I did the changes regarding your review but I also added the option on more fields.

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**